components/package-template/src/sbin/admin-tools/archive-manager.sh

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env bash
```

Comment:
we should
```
set -o errexit
set -o nounset
set -o pipefail
```
or
```
set -eu
set -o pipefail
```

so the scripts early exit on any error or on any unset variables, and make a pipeline fail if any command in it fails

@coderabbitai create an issue to track the improvement